lib/src/test/java/io/ably/lib/test/rest/RestAuthAttributeTest.java

```diff
@@ -0,0 +1,103 @@
+package io.ably.lib.test.rest;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import io.ably.lib.rest.AblyRest;
+import io.ably.lib.test.common.Setup;
+import io.ably.lib.types.AblyException;
+import io.ably.lib.types.Capability;
+import io.ably.lib.types.ClientOptions;
+
+import static io.ably.lib.rest.Auth.AuthOptions;
+import static io.ably.lib.rest.Auth.TokenCallback;
+import static io.ably.lib.rest.Auth.TokenDetails;
+import static io.ably.lib.rest.Auth.TokenParams;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.fail;
+
+/**
+ * Created by VOstopolets on 9/3/16.
+ */
+public class RestAuthAttributeTest {
+
+	private static AblyRest ably;
+
+	@BeforeClass
+	public static void setUpBeforeClass() throws Exception {
+		Setup.TestVars testVars = Setup.getTestVars();
+		ClientOptions opts = testVars.createOptions(testVars.keys[0].keyStr);
+		opts.clientId = "defaultClientId";
+		ably = new AblyRest(opts);
+	}
+
+	/**
+	 * Stores the AuthOptions and TokenParams arguments as defaults
+	 * for subsequent authorisations with the exception of the attributes:
+	 * {@link TokenParams#timestamp}, {@link AuthOptions#queryTime}, {@link AuthOptions#force}
```

Comment:
How are you asserting that `timestamp`,`queryTime` and `force` are not used as defaults for the next request?